README.md

```diff
@@ -65,6 +65,16 @@ We may collect a variety of your preferences:
 -   Window geometry
 -   Overlay stylesheet
 -   Default unit schema
+-   Selected sketcher settings:
+	- Dimensioning tool preference
+	- On-View Parameters preference
+	- If Unified Coincident and PointOnObject constraint is enabled
+	- If Auto horizontal / vertical tool is enabled
+- Panel (Task View, Tree View etc) settings:
+	- If panel is enabled
+	- If it is placed in overlay mode, if so where
+	- If transparency is enable
+	- Width/Height of the panel
```

Comment:
This is missing the workbenches enabled/disabled/default, the navigation style stuff, and the toolbar icon size.

Reply:
Ah yes, I forgot about the readme, fixed!